zpages/templates.go

```diff
@@ -116,3 +114,83 @@ func traceIDFormatter(r traceRow) template.HTML {
 func even(x int) bool {
 	return x%2 == 0
```

Comment:
Nicer to do `x & 1 == 0` although the compiler already converts it with a rewrite to the suggestion I provided.